packages/storybook/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "storybook",
+  "version": "0.0.0",
+  "license": "Apache-2.0",
+  "private": true,
+  "backstage": {
+    "role": "frontend"
+  },
```

Comment:
this is not a plugin so might not be needed

Reply:
Correct, I would add that it is unnecessary since it is not even meant to be published to NPM or a similar repository. However, TTBOMK `@backstage/cli` requires this to properly identify packages.

Reply:
I think the backstage cli is kinda dependent, we would need to make an explicit exception for this. So.. Unless we have a conflict with the linter settings enforced by backstage CLI, let's use it 🙂 Therefore let's keep the role in here, since it's just easier 🙂